scylla-cdc/src/customer.rs

```diff
@@ -0,0 +1,113 @@
+use std::collections::HashMap;
+use scylla::frame::response::result::{ColumnSpec, CqlValue, Row};
+
+pub trait Consumer {
+    fn consume_cdc(&self, data: CDCRow);
+}
+
+pub enum OperationType {
+    PreImage = 0,
+    RowUpdate = 1,
+    RowInsert = 2,
+    RowDelete = 3,
+    PartitionDelete = 4,
+    RowRangeDelInclLeft = 5,
+    RowRangeDelExclLeft = 6,
+    RowRangeDelInclRight = 7,
+    RowRangeDelExclRight = 8,
+    PostImage = 9,
```

Comment:
I know this comment is redundant at this point after @piodul's suggestion to use num_enum but for the record:
you don't have to be explicit with integer values here, they are assigned this way by default.